### PR TITLE
HG 604 Set custom var 8 at track page view time and also check if it is a main page or not

### PR DESCRIPTION
**Pull request by kenkouot**
See title.

### CODE REVIEW

File: front/scripts/mercury/modules/Trackers/GoogleAnalytics.ts [hunk 1]
```diff
@@ -54,7 +54,6 @@ module Mercury.Modules.Trackers {
 				[prefix + '_setCustomVar', 4, 'Skin', 'mercury', 3],
 				// TODO: Currently the only login status is 'anon', in the future 'user' may be an option
 				[prefix + '_setCustomVar', 5, 'LoginStatus', 'anon', 3],
```
Comment (lizlux) [inline front/scripts/mercury/modules/Trackers/GoogleAnalytics.ts]: @kenkouot, is this line something we need to make sure gets changed when we enable logged in state on mercury?

File: front/scripts/mercury/modules/Trackers/GoogleAnalytics.ts [hunk 2]
```diff
@@ -105,6 +105,16 @@ module Mercury.Modules.Trackers {
 		 * Tracks the current page view
 		 */
 		trackPageView (): void {
+			var mainPageTitle = Mercury.wiki.mainPageTitle,
+				isMainPage = window.location.pathname.split('/').indexOf(mainPageTitle),
+				pageType = ['_setCustomVar', 8, 'PageType', isMainPage ? 'home' : 'article', 3];
```
Comment (lizlux) [inline front/scripts/mercury/modules/Trackers/GoogleAnalytics.ts]: are we sure we don't need a prefix before `_setCustomVar`?

File: front/scripts/mercury/modules/Trackers/GoogleAnalytics.ts [hunk 3]
```diff
@@ -105,6 +105,16 @@ module Mercury.Modules.Trackers {
...
+
+			this.queue.push(pageType),
+
+			// Set custom var in ad account as well
+			pageType[0] = this.accounts[this.accountAds].prefix + '.' + pageType[0];
```
Comment (lizlux) [inline front/scripts/mercury/modules/Trackers/GoogleAnalytics.ts]: is there a reason we're not using the same prefix getting scheme as above? (just wondering, I'm new to this code)

Comment (kenkouot) [inline front/scripts/mercury/modules/Trackers/GoogleAnalytics.ts]: Answered this offline!

**Comment by lizlux** [comment]
LGTM

**Pull request merged**